platforms/documentation/docs-asciidoctor-extensions-base/src/main/resources/clipboard.js

```diff
@@ -0,0 +1,42 @@
+/* Custom clipboard JSS code */
```

Comment:
```suggestion
/* Custom JS on top of external clipboard.js */
```

Let's make it clear what this builds on